alerts/system_alerts.libsonnet

```diff
@@ -174,6 +187,19 @@ local utils = import 'utils.libsonnet';
               message: 'A client certificate used to authenticate to the apiserver is expiring in less than %s.' % (utils.humanizeSeconds($._config.certExpirationCriticalSeconds)),
             },
           },
+          {
+            alert: 'KubeletPlegIntervalHigh',
+            expr: |||
+              max(kubelet_pleg_relist_interval_seconds{quantile="0.99"}) by (node) > %(kubeletPlegIntervalWarningSeconds)s
```

Comment:
The metric is a histrogram and called `kubelet_pleg_relist_duration_seconds`. I don't believe this will work as is.

Reply:
I'll find some time to get a local 1.14+ cluster running and clean this up.  Hadn't noticed that the type changed from summary to histogram to go along with the changing of the metric name.

Reply:
The latest commit attempts to address the change of the metric to a histogram.  I also switched from using the `interval` to `duration`, which seems clearer to me.